css/style.css

```diff
@@ -0,0 +1,17 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+.title1{
+    text-align: center;
+}
+label{
+    display: block;
+}
+body{
+    background-color: palegreen;
+}
+form{
+    margin: 0 30%;
```

Comment:
Nice styling 💯 

I think it would be awesome if you could add an indentation of 2 space as recommended, this would make you styling readable and professional.

Reply:
Great work @emarkees, now you just need to add one more review comment and you are good to go since the project requirements expects you to give at least 2 review comments. :+1: